### PR TITLE
FOGL-7062 Add support to sqlitelb to create readings table

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Used when we set sqlitelb as the readings plugin while keeping sqlite as the configuration plugin

### CODE REVIEW

**Review by pintomax — APPROVED** [review]

File: C/plugins/storage/sqlitelb/common/connection.cpp [hunk 1]
```diff
@@ -587,6 +606,151 @@ Connection::Connection()
 		//Release sqlStmt buffer
 		delete[] sqlReadingsStmt;
 
+		if (initialiseReadings)
```
Comment (deepanshu-yadav) [inline C/plugins/storage/sqlitelb/common/connection.cpp:609]: Why are doing this isn't init.sql of sqlitelb storage does this?  I am suggesting the following  thing 

when storage.sh fetches the required storage plugin the storage.cpp should also return readings plugin along with storage plugin name. The script will execute (storage.sh) will execute corresponding shell scripts of the required plugins. 

For example if we give storage plugin sqlite and readings plugin sqlitelb then it should execute the shell script  
 sqlite.sh  and sqlitelb.sh respectively.

File: Makefile [hunk 2]
```diff
@@ -25,7 +25,7 @@ ifneq ("$(PLATFORM_RH)","")
 else
 	PIP_INSTALL_REQUIREMENTS := pip3 install -Ir
 	PYTHON_BUILD_PACKAGE = python3 setup.py build -b ../$(PYTHON_BUILD_DIR)
-	CMAKE := cmake
+	CMAKE := cmake -DCMAKE_BUILD_TYPE=Debug
 endif
```
Comment (deepanshu-yadav) [inline Makefile:29]: Should be removed afterwards i suppose?

File: C/plugins/storage/sqlitelb/common/connection.cpp (hunk 1 above)
Comment (MarkRiddoch) [reply C/plugins/storage/sqlitelb/common/connection.cpp:609]: It doesn't, but it assumes the sqlitelb is being used for both config and readings, that is not the case the is solving. Tis is for when we switch sqlitelb in for just the readings. Ideally we want to run the init_readings.ql, but there are issues with that, so this is a short term fix until we get a chance to rethink the initialisation. The same thing holds true if Postgres is used for readings only.

**Review by RVerhoeff — APPROVED** [review]

**Comment by praveen-garg** [comment]
@MarkRiddoch This is waiting for some other fixes?

**Review by pintomax — APPROVED** [review]

**Pull request merged**